doc/source/reference/swig.interface-file.rst

```diff
@@ -1,4 +1,4 @@
-Numpy.i: a SWIG Interface File for NumPy
+NumPy.i: a SWIG Interface File for NumPy
```

Comment:
This should be changed to all lower case, numpy.i, which is the actual name of the file it refers to.